src/App.js

```diff
@@ -1,19 +1,156 @@
 import React from 'react';
+
+import { Button, Alert, Col, Container, Form, Row } from 'react-bootstrap';
+
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
+
 import './App.css';
 
-import users from './api/users';
+const preparedTodos = todosFromServer.map(
+  item => ({
+    ...item,
+    user: usersFromServer.find(person => (person.id === item.userId)),
+  }),
```

Comment:
```suggestion
  todo => ({
    ...todo,
    user: usersFromServer.find(person => (person.id === todo.userId)),
  }),
```
"лёгким движением руки тудушки превращаються в элегантные итемы"

Нє-а, не та історія)